lib/stat_tracker.rb

```diff
@@ -21,4 +21,31 @@ def load_team_data
     @team_collection = TeamCollection.new(@teams_file_path)
   end
 
+  def highest_total_score
+    game_collection.games.map do |game|
+      game.home_goals + game.away_goals
+    end.max
+  end
+
+  def biggest_blowout
+    game_collection.games.map do |game|
+      Math.sqrt((game.home_goals - game.away_goals)**2).to_i
+    end.max
+  end
+
+  def percentage_ties
+    ties = game_collection.games.find_all do |game|
+      game.home_goals == game.away_goals
+    end.length
+    (ties / game_collection.games.length.to_f).round(2)
+  end
+
+  def count_of_games_by_season
```

Comment:
Would a reduce method improve this?

Reply:
I went with hash.new to set a default value.  I don't think Reduce would be too much more helpful and this was easier.  Maybe a refactor later though!